### DISCUSSION TITLE
Face Attention Result

### DISCUSSION THREAD

**Discussion by ilkerkocatepe** [vladmandic/human#457] [Q&A] [answered]
Firstly I really appreciate this project, thank you for wonderful work.
I'm trying demo applications in my local machine and using nodejs backend module. I just wanted to see attention result for detected faces. Object, body, hand statuses are disabled, and mesh, iris, emotion, attention is enabled in config. (The file is /demo/nodejs
/process-folder.js)
When I tried to log the data (result.face[i].attention) is undifined. How can I tackle this issue?

**Comment by vladmandic** [comment, marked as answer]
face attention is not enabled for nodejs as required tensorflow operations that connect multiple models into single attention are not available in tensorflow bin used by tfjs in nodejs, only in webgl backend used by tfjs for browsers.

you can see results of automated tests which try to test all models on all backends in `test/test.log`
